examples/audio-classification/run_audio_classification.py

```diff
@@ -46,8 +46,8 @@ def check_optimum_habana_min_version(*a, **b):
 logger = logging.getLogger(__name__)
 
 # Will error if the minimal version of Transformers and Optimum Habana are not installed. Remove at your own risks.
-check_min_version("4.40.0")
-check_optimum_habana_min_version("1.11.0")
+check_min_version("4.43.0")
+check_optimum_habana_min_version("1.12.0")
```

Comment:
Hi, I am trying to run this test with these versions but encountered the following error:
```bash
ERROR: Cannot install optimum-habana==1.12.1 and transformers==4.43.0 because these package versions have conflicting dependencies.\n#8 20.54 \n#8 20.54 The conflict is caused by:\n#8 20.54     The user requested transformers==4.43.0\n#8 20.54     optimum-habana 1.12.1 depends on transformers<4.41.0 and >=4.40.0
```
Can someone please point me to the PR that corrected or if there is any documentation to fix this? 
Thanks!

Reply:
You should pip install optimimum habana from git, because this fork is ahead of 1.12.1, and have different dependencies, and this should be listed as 1.13.0.dev as the version number.

Reply:
Yeah it should be `check_optimum_habana_min_version("1.13.0.dev0")`, I'll add a script to do that automatically.

Reply:
> You should pip install optimimum habana from git, because this fork is ahead of 1.12.1, and have different dependencies, and this should be listed as 1.13.0.dev as the version number.

Ok great, Thanks for the info. So, currently, I am using `pip install git+https://github.com/huggingface/optimum-habana.git@{{ optimum_habana_version }}`. Would using `pip install git+https://github.com/huggingface/optimum-habana.git` to use the latest version that is compatible with transformers==4.43.0 work for future transformers versions?

Reply:
It may not work with future Transformers versions as there might be changes that are not compatible with what we do here in Optimum Habana.
In the coming weeks, I will open a new branch and try to maintain it so that new Transformers releases are supported but with potential perf regressions (which will be solved once it comes to the main branch).

Reply:
Thanks @regisss, that would be helpful.
Until then, I will be using transformers==4.43.0 while doing `pip install git+https://github.com/huggingface/optimum-habana.git`. It worked for me yesterday without specifying the optimum-habana version.
Can you please confirm if this would work only for 4.43.0, please?
For future transformers' version - I'll be keep an eye out for the new branch you mentioned.

Reply:
Yes, the main branch should work for Transformers 4.43.x till the next time we align the lib with a new version of Transformers.